cdap-cli/src/main/java/co/cask/cdap/cli/util/ArgumentParser.java

```diff
@@ -56,6 +56,8 @@
         String key = token.substring(0, firstEquals);
         String value = token.substring(firstEquals + 1, token.length());
         result.put(extractValue(key), extractValue(value));
+      } else {
+        throw new IllegalArgumentException(description + " must be of the form 'key1=val1 key2=val2'");
```

Comment:
But looking at your output: would we not to really say:
```
cdap> start mapreduce myApp.myMR "arg1=1000 arg2"
Error: runtime-args must be of the form "key1=val1 key2=val2"
```
using double-quotes instead of single-quotes?